client/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "jsmp-client",
+  "version": "1.0.0",
+  "description": "javascript mentoring program 2020",
+  "main": "index.js",
+  "scripts": {
+    "start": "webpack-dev-server --open --config webpack.dev.js",
+    "build": "webpack --config webpack.prod.js",
+    "test": "jest",
+    "lint": "eslint src --ext 'js,ts'"
+  },
+  "author": "Nataliia Mogylchuk",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/nmogylchuk/js-mentoring-program"
+  },
+  "keywords": [
+    "typescript",
+    "babel",
+    "webpack"
+  ],
+  "license": "ISC",
+  "devDependencies": {
+    "@babel/cli": "^7.12.1",
+    "@babel/core": "^7.12.3",
+    "@babel/preset-env": "^7.12.1",
+    "@babel/preset-typescript": "^7.12.1",
+    "@types/jest": "^26.0.15",
+    "@typescript-eslint/eslint-plugin": "^4.5.0",
+    "@typescript-eslint/parser": "^4.5.0",
+    "@webpack-cli/serve": "^1.0.1",
+    "babel": "^6.23.0",
+    "babel-loader": "^8.1.0",
+    "babel-preset-es2015": "^6.24.1",
+    "css-loader": "^5.0.0",
+    "eslint": "^7.11.0",
+    "eslint-config-airbnb-base": "^14.2.0",
+    "eslint-config-prettier": "^6.14.0",
+    "eslint-loader": "^4.0.2",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-prettier": "^3.1.4",
+    "eslint-webpack-plugin": "^2.1.0",
+    "identity-obj-proxy": "^3.0.0",
+    "jest": "^26.6.1",
+    "pre-commit": "^1.2.2",
+    "prettier": "2.1.2",
+    "sass": "^1.27.0",
+    "sass-loader": "^10.0.4",
+    "style-loader": "^2.0.0",
+    "ts-jest": "^26.4.1",
+    "ts-loader": "^8.0.6",
+    "typescript": "^4.0.3",
+    "webpack": "^5.2.0",
+    "webpack-cli": "^3.3.4",
+    "webpack-dev-server": "^3.11.0",
+    "webpack-merge": "^5.2.0"
+  },
+  "dependencies": {
+    "clean-webpack-plugin": "^3.0.0",
+    "html-webpack-plugin": "^4.5.0",
+    "jest-transform-css": "^2.1.0",
+    "webpack-node-externals": "^2.5.2"
+  },
+  "pre-commit": [
```

Comment:
Looks like it doesn't work. Could you try to add pre-commit hook by this article https://elijahmanor.com/blog/npm-precommit-scripts ?